app/views/admin/notes/_show.html.slim

```diff
@@ -0,0 +1,23 @@
+header.small
+  .post-info
+    .item
+      i.fa.fa-calendar-o>
+      = l note.created_at.to_date
+    .item
+      a href=admin_person_path(note.author)
+        i.fa.fa-user>
+        = note.author.name
```

Comment:
Technically you should be using delegators for this kind of thing `e.g. author_name` so you don't violate LoD. It's a good habit to get in.